config/db.js

```diff
@@ -3,22 +3,12 @@ const { db: { uri: databaseURI } } = require('./index');
 const logger = require('../helpers/logger');
 const { gracefulShutdown } = require('../helpers/util');
 
-mongoose.connect(databaseURI, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-  useFindAndModify: false,
-  useCreateIndex: true,
-});
-
-const conn = mongoose.connection;
+const conn = async () => mongoose.connect(databaseURI);
```

Comment:
You're missing an `await` here @Andrew-Paystack

Reply:
I think you should also consider putting this in a try catch block, in case there are errors when attempting to connect to the DB